packages/discovery-provider/src/tasks/entity_manager/entities/user.py

```diff
@@ -336,6 +336,15 @@ def update_user_metadata(
                 continue
             setattr(user_record, key, metadata[key])
 
+    # Ensure verified social handle is same as audius handle
+    if user_record.is_verified:
+        if user_record.verified_with_twitter:
+            user_record.twitter_handle = user_record.handle
+        elif user_record.verified_with_instagram:
```

Comment:
`if` instead of `elif` because some users are verified with more than one social platform